.buildkite/scripts/packaging/package-dra.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+BEAT_DIR=$1
+
+if [ -z "$BEAT_DIR" ]; then
+    echo "Error: Beat directory must be specified."
+    exit 1
+fi
+
+echo "Packaging : $BEAT_DIR"
```

Comment:
What about prefixing with somethink like `~~~` or `+++` depending on if we want it to be expanded or not in BK ui.